Packs/Hackuity/Integrations/Hackuity/Hackuity.yml

```diff
@@ -0,0 +1,228 @@
+      isArray: false
+      name: trs_max
+      required: false
+      secret: false
+    - &argument_vuln_type
+      default: false
+      description: The vulnerability type (ID)
+      isArray: false
+      name: vuln_type
+      required: false
+      secret: false
+    deprecated: false
+    description: Search for findings in Hackuity.
+    name: hackuity-search-findings
+    outputs:
+    - contextPath: Hackuity.Findings.Asset.Id
+      description: The ID of the asset
+      type: string
+    - contextPath: Hackuity.Findings.Asset.Name
+      description: The name of the asset
+      type: string
+    - contextPath: Hackuity.Findings.Asset.Type
+      description: The type of the asset
+      type: string
+    - contextPath: Hackuity.Findings.Attributes
+      description: The attributes of the finding
+      type: string
+    - contextPath: Hackuity.Findings.Score.CVSS
+      description: The CVSS of the finding
+      type: string
+    - contextPath: Hackuity.Findings.Score.TRS
+      description: The TRS of the finding
+      type: string
+    - contextPath: Hackuity.Findings.Id
+      description: The ID of the finding
+      type: string
+    - contextPath: Hackuity.Findings.Status.Ignored
+      description: Whether the finding is ignored
+      type: boolean
+    - contextPath: Hackuity.Findings.Status.State
+      description: The state of the finding
+      type: string
+    - contextPath: Hackuity.Findings.Status.SubState
+      description: The sub-state of the finding
+      type: string
+    - contextPath: Hackuity.Findings.VulnType.Id
+      description: The ID of the vulnerability type
+      type: string
+    - contextPath: Hackuity.Findings.VulnType.Name
+      description: The name of the vulnerability type
+      type: string
+  - arguments:
+    - *argument_asset_name
+    - *argument_asset_type
+    - *argument_attribute
+    - *argument_cvss_min
+    - *argument_cvss_max
+    - *argument_limit
+    - *argument_trs_min
+    - *argument_trs_max
+    - *argument_vuln_type
+    deprecated: false
+    description: Search for vulndb vulnerabilities in Hackuity.
+    name: hackuity-search-vulndb-vulnerabilities
+    outputs: &outputs_vulnerabilities
+    - contextPath: Hackuity.Vulnerabilities.Attributes
+      description: The attributes of the vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.Score.CVSS
+      description: The CVSS of the vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.Score.TRS
+      description: The TRS of the vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.Description
+      description: The description of the vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.ID
+      description: The ID of the vulnerability
+      type: string
+    - contextPath: Hackuity.Vulnerabilities.Seen.First
+      description: The date of the first time the vulnerability has been seen
```

Comment:
since you have a specific format, add an example for this date

Reply:
The output date is specifically formatted with `DATE_FORMAT` which I thought was the way dates are represented inside Cortex XSOAR? e.g. `2021-03-03T07:56:07Z`

Tell me if that's not the case, so that I can change the date formatting so that is it the one used by default inside Cortex XSOAR.

Reply:
yeah its okay, just added to the description as an example, that's it